apps/aevm/src/aevm_ae_primops.erl

```diff
@@ -643,6 +645,8 @@ crypto_call(Gas, Op, _Value, Data, State) ->
             crypto_call(Gas, Op, Data, State)
     end.
 
+crypto_call(Gas, ?PRIM_CALL_CRYPTO_ECRECOVER_SECP256K1, Data, State) ->
```

Comment:
There needs to be a check (^^) that we are only allowed to run this if VM >= ?VM_AEVM_SOPHIA_4

Reply:
Thanks. I added a check.

Reply:
Not really where I'd have put it, but it works :-)